backend/workflow_manager/endpoint/destination.py

```diff
@@ -434,23 +433,22 @@ def _push_to_queue(
         Returns:
             None
         """
+        if not result:
+            return
         connector: ConnectorInstance = self.endpoint.connector_instance
         connector_settings: dict[str, Any] = connector.connector_metadata
         queue_result = {"file": file_name}
-        if error:
-            return
-        if result:
-            queue_result.update(
-                {
-                    "status": QueueResultStatus.SUCCESS,
-                    "result": result,
-                    "workflow_id": self.workflow_id,
-                    "connector_id": connector.id,
-                }
-            )
-            queue_class = QueueUtils.get_queue_class(
-                connector_id=connector.connector_id,
-                connector_settings=connector_settings,
-            )
-            # TODO: Replace queue name with document class name (tool name)
-            queue_class.enqueue(queue_name="review_queue", message=str(queue_result))
+        queue_result.update(
```

Comment:
NIT: You can create the `queue_result` dictionary directly rather than just populating it with `file_name` in line 440 and then updating it here

Reply:
done